### PR TITLE
Add documentation for accessing browser instances via strings

### PR DESCRIPTION
**Pull request by evilC**
## Proposed changes

Documents accessing Multiremote browser instances via the browser object using strings

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: website/docs/Multiremote.md [hunk 1]
```diff
@@ -151,3 +151,37 @@ assert.true(
 In this example, the `myFirefoxBrowser` instance will start waiting on a message once the `myChromeBrowser` instance has clicked on `#send` button.
 
 Multiremote makes it easy and convenient to control multiple browsers, whether you want them doing the same thing in parallel, or different things in concert.
+
+## Accessing browser instances using strings via the browser object
+In addition to accessing browser instance via their global variables (eg `myChromeBrowser`, `myFirefoxBrowser`), you can also access them via the `browser` object - eg `browser["myChromeBrowser"]` or `browser["myFirefoxBrowser"]`
```
Comment (christian-bromann) [inline website/docs/Multiremote.md:156]:
```suggestion
In addition to accessing browser instance via their global variables (eg `myChromeBrowser`, `myFirefoxBrowser`), you can also access them via the `browser` object - eg `browser["myChromeBrowser"]` or `browser["myFirefoxBrowser"]`.
```

File: website/docs/Multiremote.md [hunk 2]
```diff
@@ -151,3 +151,37 @@ assert.true(
 In this example, the `myFirefoxBrowser` instance will start waiting on a message once the `myChromeBrowser` instance has clicked on `#send` button.
 
 Multiremote makes it easy and convenient to control multiple browsers, whether you want them doing the same thing in parallel, or different things in concert.
+
+## Accessing browser instances using strings via the browser object
+In addition to accessing browser instance via their global variables (eg `myChromeBrowser`, `myFirefoxBrowser`), you can also access them via the `browser` object - eg `browser["myChromeBrowser"]` or `browser["myFirefoxBrowser"]`  
+This is especially useful when writing re-usable test steps that can be performed in either browser:  
+eg
```
Comment (christian-bromann) [inline website/docs/Multiremote.md:158]:
```suggestion
This is especially useful when writing re-usable test steps that can be performed in either browser, e.g.:
```

File: website/docs/Multiremote.md [hunk 3]
```diff
@@ -151,3 +151,37 @@ assert.true(
...
+        }
+    }
+```
+
+Cucumber file:
+```js
+When User A types a message into the chat
+```
+
+Step definition file:
```
Comment (christian-bromann) [inline website/docs/Multiremote.md:181]:
```suggestion
Cucumber file:
    ```feature
    When User A types a message into the chat
    ```

Step definition file:
```

File: website/docs/Multiremote.md [hunk 4]
```diff
@@ -151,3 +151,37 @@ assert.true(
...
+```
```
Comment (christian-bromann) [inline website/docs/Multiremote.md:182]:
```suggestion
```js
```

File: website/docs/Multiremote.md [hunk 5]
```diff
@@ -151,3 +151,37 @@ assert.true(
...
+When(/^User (.) types a message into the chat/, (userId) => {
+    browser["user" userId].$('#message').setValue('Hi, I am Chrome')
+    browser["user" userId].$('#send').click()
```
Comment (christian-bromann) [inline website/docs/Multiremote.md:185]:
```suggestion
    browser[`user${userId}`].$('#message').setValue('Hi, I am Chrome')
    browser[`user${userId}`].$('#send').click()
```

**Review by christian-bromann — CHANGES_REQUESTED** [review]

File: website/docs/Multiremote.md [hunk 6]
```diff
@@ -151,3 +151,36 @@ assert.true(
 In this example, the `myFirefoxBrowser` instance will start waiting on a message once the `myChromeBrowser` instance has clicked on `#send` button.
 
 Multiremote makes it easy and convenient to control multiple browsers, whether you want them doing the same thing in parallel, or different things in concert.
+
+## Accessing browser instances using strings via the browser object
+In addition to accessing browser instance via their global variables (eg `myChromeBrowser`, `myFirefoxBrowser`), you can also access them via the `browser` object - eg `browser["myChromeBrowser"]` or `browser["myFirefoxBrowser"]`.
```
Comment (christian-bromann) [inline website/docs/Multiremote.md:156]:
```suggestion
In addition to accessing the browser instance via their global variables (e.g. `myChromeBrowser`, `myFirefoxBrowser`), you can also access them via the `browser` object, e.g. `browser["myChromeBrowser"]` or `browser["myFirefoxBrowser"]`.
```

**Review by christian-bromann — APPROVED** [review]
👍  thanks!

**Comment by christian-bromann** [comment]
@evilC before I can merge can you please sign the CLA?

File: website/docs/Multiremote.md [hunk 7]
```diff
@@ -151,3 +151,36 @@ assert.true(
 In this example, the `myFirefoxBrowser` instance will start waiting on a message once the `myChromeBrowser` instance has clicked on `#send` button.
 
 Multiremote makes it easy and convenient to control multiple browsers, whether you want them doing the same thing in parallel, or different things in concert.
+
+## Accessing browser instances using strings via the browser object
+In addition to accessing the browser instance via their global variables (e.g. `myChromeBrowser`, `myFirefoxBrowser`), you can also access them via the `browser` object, e.g. `browser["myChromeBrowser"]` or `browser["myFirefoxBrowser"]`.
+This is especially useful when writing re-usable test steps that can be performed in either browser, e.g.:
```
Comment (christian-bromann) [inline website/docs/Multiremote.md:157]:
```suggestion
In addition to accessing the browser instance via their global variables (e.g. `myChromeBrowser`, `myFirefoxBrowser`), you can also access them via the `browser` object, e.g. `browser["myChromeBrowser"]` or `browser["myFirefoxBrowser"]`. You can get a list of all your instances via `browser.instances`. This is especially useful when writing re-usable test steps that can be performed in either browser, e.g.:
```

**Comment by evilC** [comment]
Looks like something went slightly wrong:
image

**Pull request merged**